lib/config/index.js

```diff
@@ -76,6 +76,11 @@ function getDefaultConfig() {
         throw new GeminiError('No config found');
     }
 
+    if (!/\.yml$/.test(configFile)) {
```

Comment:
Sure thing. I initially left `yml` out to prevent long paths in case there was an error later and it popped up in the stack trace.